src/game.cpp

```diff
@@ -9792,17 +9804,24 @@ bool game::handle_liquid( item &liquid, item * const source, const int radius,
         menu.text = string_format( _( "What to do with the %s from %s?" ), liquid_name.c_str(), m.name( *source_pos ).c_str() );
     } else if( source_veh != nullptr ) {
         menu.text = string_format( _( "What to do with the %s from the %s?" ), liquid_name.c_str(), source_veh->name.c_str() );
-    } else {
+
+	} else if( source_mon != nullptr ) {
+	    menu.text = string_format( _( "What to do with the %s from the %s?" ), liquid_name.c_str(),
+	                               source_mon->disp_name().c_str() );
+	} else {
         menu.text = string_format( _( "What to do with the %s?" ), liquid_name.c_str() );
     }
     std::vector<std::function<void()>> actions;
-
-    if( u.can_consume( liquid ) ) {
-        menu.addentry( -1, true, 'e', _( "Consume it" ) );
-        actions.emplace_back( [&]() {
-            // consume_item already consumes moves.
-            u.consume_item( liquid );
-        } );
+    // Small check to avoid players sucking milk directly from the animal.
```

Comment:
I asked if the player should be able to suck milk directly from the udders, and the general consensus on the IRC channel was to disabled it.